python-threatexchange/threatexchange/signal_type/tlsh_pdf.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+"""
+Wrapper around the pdf signal type.
+"""
+
+import pathlib
+import typing as t
+import warnings
+from io import StringIO
+import tlsh
+from pdfminer.converter import TextConverter
+from pdfminer.layout import LAParams
+from pdfminer.pdfdocument import PDFDocument
+from pdfminer.pdfinterp import PDFResourceManager, PDFPageInterpreter
+from pdfminer.pdfpage import PDFPage
+from pdfminer.pdfparser import PDFParser
+
+from ..descriptor import SimpleDescriptorRollup, ThreatDescriptor
+from . import signal_base
+
+TLSH_CONFIDENT_MATCH_THRESHOLD = 30
+EXPECT_TLSH_HASH_LENGTH = 72
+""" 
+This is a new signal type not found on ThreatExchange, in order to test the implementation we
+override this set of hashes we want to match with here.
+ 
+ToDo: @BarrettOlson: make this override cleaner and configurable by signal type in the future 
+"""
+TEMP_MATCH_IMPLEMNTATION_CHECK_DB = [
+    [
+        "T145B2859FE708266211A3026277C7AEE5FF76402C636AD5BA2C2CC11C23A1F2957773D5",
+        ["test", 1],
+    ]
+]
+
+
+class TLSHSignal(
+    signal_base.SimpleSignalType, signal_base.FileHasher, signal_base.BytesHasher
+):
+    """
+    Simple signal type for PDFs using TLSH
+
+    Extracts the text from a given pdf using pdfminer.six and hashes it with TLSH
+
+    """
+
+    INDICATOR_TYPE = "HASH_PDF"
```

Comment:
Since the INDICATOR doesn't exist yet in TE we can probably remove this line.
If you want to keep it or end up needing it `'HASH_TSLH'` would be preferred